app/proc/store_test.go

```
@@ -71,9 +71,9 @@ func TestSaveIfItemIsExists(t *testing.T) {
 	item := feed.Item{
 		PubDate: pubDate,
 	}
-	created, err := boltDB.Save("radio-t", item)
+	boltDB.Save("radio-t", item) //nolint:gosec
```

in such tests ignoring errors not the best idea. 

```
_, err := boltDB.Save("radio-t", item) //nolint:gosec
require.NoError(t, err)
```